src/frontend/hooks/hasStatus.ts

```diff
@@ -24,9 +27,25 @@ export function hasStatus(
   const {
     thirdPartyManagedApp = undefined,
     is_installed,
-    runner,
+    runner = 'sideload',
```

Comment:
I'm confused about this change, why we default to `sideload` for this custom hook?

Reply:
If I dont change this I need the gameinfo to get the status. The problem is that there is no gameinfo yet since Steam was not installed. It is a small change that won't change anything that uses this hook. I tested already all the states on the gamecard and gamepage. It is just a default and the other hooks passes the gameinfo so its totally safe to do that.